scripts/system/libraries/entitySelectionTool.js

```diff
@@ -205,7 +205,7 @@ SelectionManager = (function() {
 
         for (var i = 0; i < listeners.length; i++) {
             try {
-                listeners[i]();
+                listeners[i](selectionUpdated === true);
```

Comment:
It's just making sure the parameter is a boolean. If you don't pass it it will be `undefined`.